eth/backend.go

```diff
@@ -266,20 +266,6 @@ func New(ctx *node.ServiceContext, config *Config) (*Ethereum, error) {
 	}
 	eth.APIBackend.gpo = gasprice.NewOracle(eth.APIBackend, gpoParams)
 
-	eth.dialCandiates, err = eth.setupDiscovery(&ctx.Config.P2P)
```

Comment:
Why is next block being removed?

Reply:
I didn't remove it. it was removed from `master`.

you can see that is not present anymore: https://github.com/ConsenSys/quorum/blob/master/eth/backend.go#L267

Reply:
however, is actually present in geth: https://github.com/ethereum/go-ethereum/blob/v1.9.11/eth/backend.go#L225.

so then should we keep?